bin/test

```diff
@@ -6,10 +6,10 @@ const fs = require('fs');
 
 const downloadAndUnzipVSCode = require('vscode-test').downloadAndUnzipVSCode;
 
-const version = process.env.CODE_VERSION || '*';
+const version = process.env.CODE_VERSION || '<latest>';
 const isInsiders = version === 'insiders';
 
-const testRunFolder = path.join('.vscode-test', isInsiders ? 'insiders' : 'stable');
+const testRunFolder = path.join('.vscode-test', isInsiders ? 'insiders' : `vscode-${version}`);
```

Comment:
If the version is `insiders`, it should be downloaded to `.vscode-test/vscode-insiders`.